lib/rules/handle-done-callback.js

```diff
@@ -1,18 +1,43 @@
 'use strict';
 
-var _ = require('lodash');
+var _ = require('lodash'),
+    estraverse = require('estraverse');
 
+/*eslint-disable max-statements*/
 module.exports = function (context) {
     var possibleAsyncFunctionNames = [
-        'it',
-        'it.only',
-        'test',
-        'test.only',
-        'before',
-        'after',
-        'beforeEach',
-        'afterEach'
-    ];
+            'it',
+            'it.only',
+            'test',
+            'test.only',
+            'before',
+            'after',
+            'beforeEach',
+            'afterEach'
+        ],
+        rootNode;
+
+    function getNodeByRangeIndex(index) {
+        var result = null;
+
+        estraverse.traverse(rootNode, {
+            enter: function (node, parent) {
+                if (node.range[0] <= index && index < node.range[1]) {
+                    result = node;
+                    result.parent = parent;
+                } else {
+                    this.skip();
+                }
+            },
+            leave: function (node) {
+                if (node === result) {
+                    this.break();
+                }
+            }
+        });
+
+        return result;
+    }
```

Comment:
This function is copied and slightly modified from eslint. This can be removed with the next eslint version which will include this changes.